tests/v2/test_waku_rln_relay.nim

```diff
@@ -495,44 +246,44 @@ suite "Waku rln relay":
     check member_is_added
 
     # read the Merkle Tree root after insertion
-    var 
-      root2 {.noinit.} : Buffer = Buffer()
+    var
+      root2 {.noinit.}: Buffer = Buffer()
       rootPtr2 = addr(root2)
       get_root_successful2 = get_root(rlnInstance.value, rootPtr2)
     check:
       get_root_successful2
       root2.len == 32
 
-    # delete the first member 
+    # delete the first member
     var deleted_member_index = MembershipIndex(0)
     let deletion_success = delete_member(rlnInstance.value, deleted_member_index)
     check deletion_success
```

Comment:
Thanks for the suggestion, though wondering whether it is a new convention to split the `check` from the condition? otherwise, the current version works, why the split?

Reply:
For these, I had in mind the suggestion provided [here](https://github.com/status-im/nim-waku/pull/931#discussion_r841574139), to have some consistency among single-check tests. While there there were inconsistencies between single and two-lines single-checks (like the ones in the suggestion), here I see you used either `check: condition` and `check condition`. I personally think the most versatile version which cover all these cases for single/multi checks is 

```
check:
  condition(s)
```

since it further allows to easily add new checks when/if needed by future PRs. But this is minor suggestion and we might indeed agree on some common convention.

Reply:
I see, I think it is an implicit agreement not explicit, all the rln-relay tests use one-line checks to assert a single condition.  I have split them now https://github.com/status-im/nwaku/pull/941/commits/f9417a30cab96e0ec4b035a434aaa56b42081a54, though it would make sense to consolidate all such internal and implicit conventions to make sure everyone is on the same page.